api/institutions/views.py

```diff
@@ -453,13 +463,20 @@ def _make_elasticsearch_results_filterable(self, search, **kwargs) -> MockQuerys
     def get_queryset(self):
         return self.get_queryset_from_request()
 
+
 class InstitutionDepartmentList(InstitutionImpactList):
     view_name = 'institution-department-metrics'
 
     serializer_class = InstitutionDepartmentMetricsSerializer
+    renderer_classes = tuple(api_settings.DEFAULT_RENDERER_CLASSES) + (InstitutionDepartmentMetricsCSVRenderer, )
 
     ordering = ('-number_of_users', 'name',)
 
+    def is_csv_export(self):
```

Comment:
It could be based off of the request header but the renderer class can also change if `?format=csv` is provided so I think basing it off of the set renderer class is the best course of action so that it is in-line with the content negotiation results.